Docker/README.md

```diff
@@ -157,6 +125,42 @@ sudo docker rm freshrss_old
 ```
 
 
+## [Docker tags](https://hub.docker.com/r/freshrss/freshrss/tags)
+The tags correspond to FreshRSS branches and versions:
+* `:latest` (default) is the `master` branch, more stable
+* `:dev` is the `dev` branch, rolling release
+* `:x.y.z` are specific FreshRSS releases
+
+### Linux: Ubuntu vs. Alpine
+Our default image is based on [Ubuntu](https://www.ubuntu.com/server). We offer an alternative based on [Alpine](https://alpinelinux.org/) (with the `-alpine` tag suffix).
+In [our tests](https://github.com/FreshRSS/FreshRSS/pull/2205), Ubuntu is ~3 times faster,
+while Alpine is ~2.5 times [smaller on disk](https://hub.docker.com/r/freshrss/freshrss/tags) (and much faster to build).
+
+
+## Optional: Build Docker image of FreshRSS
+Optional, as a *less recent* online image can be automatically fetched,
+but online images are not available for as many platforms (e.g. Raspberry Pi / ARM) as if you build yourself.
```

Comment:
Building your own Docker image is optional because *less recent* online images can be fetched automatically. Please keep in mind that prebuilt images are only available for a limited number of platforms.

Reply:
Done :-)